cookiecutter.json

```diff
@@ -0,0 +1,10 @@
+{
+    "author_name": "",
```

Comment:
This file should have some cookiecutter bits.  cf https://github.com/jupyter/widget-cookiecutter/blob/master/%7B%7Bcookiecutter.github_project_name%7D%7D/setup.py